src/MakingSense.AspNetCore.Documentation/BaseDocumentHandler.cs

```diff
@@ -20,10 +20,25 @@ public abstract class BaseDocumentHandler : IDocumentHandler
 
 		public int StatusCode => StatusCodes.Status200OK;
 
-		public BaseDocumentHandler(IFileProvider fileProvider, string subpath)
+		public BaseDocumentHandler(IFileProvider fileProvider, string subpath, List<string> languages)
+		{
+			foreach (var lang in languages)
+			{
+				SetFileInfo(fileProvider, subpath, $".{lang}");
+
+				if(FileInfo != null)
+				{
+					return;
+				}
+			}
+
+			SetFileInfo(fileProvider, subpath);
+		}
+
+		private void SetFileInfo(IFileProvider fileProvider, string subpath, string langPrefix = "")
```

Comment:
I am not comfortable with this method. What do you think about a function that returns a _FileInfo object_ in place of set _FileInfo field_ value?